readme.md

```diff
@@ -18,15 +18,27 @@ $ npm install --global osx-wallpaper
 [Download the binary](https://github.com/sindresorhus/osx-wallpaper/releases/latest) and put it in `/usr/local/bin`.
 
 
-## Usage
+## Binary usage
 
 ```sh
-# set
+# Set wallpaper on main screen
 wallpaper unicorn.jpg
 
-# get
+# List connected screens (screen ID:screen name)
+wallpaper --screeninfo
+> 69732482:Color LCD
+> 188845747:SM2333T
+
+# Set wallpaper on specific screen (replace screenid with your screen ID)
+wallpaper screenid unicorn2.jpg
```

Comment:
I'm not sold on using screenid. I was hoping we could do something simple, like just referring to them as screen numbers. Eg, main screen is `1`, the next is `2`, etc. That would also make `--screeninfo` moot. Thoughts?

// @gpapin @dorian-marchal @kevva @arthurvr